docs/tempo/website/configuration/parquet.md

```diff
@@ -53,24 +53,24 @@ storage:
 
 ## Parquet configuration parameters
 
-Some parameters in the Tempo configuration are specific to Parquet.  
+Some parameters in the Tempo configuration are specific to Parquet.
 For more information, refer to the [storage configuration documentation](https://grafana.com/docs/tempo/latest/configuration/#storage).
```

Comment:
This is an old link format. It's currently a relref, which makes me think your updates were made to an out-of-date file.